vic/vic_run/src/lakes.eb.c

```diff
@@ -25,6 +25,7 @@
  *****************************************************************************/
 
 #include <vic_run.h>
+#include <vic_driver_shared_all.h>
```

Comment:
I just noticed this in my follow up review.  We do not want `vic_driver_shared.h` anywhere in `vic_run`.  If we need to move a function from `drivers/shared_all` to `vic_run`, we can do that but this include needs to go.